code/modules/mob/living/simple_animal/abnormality/he/drifting_fox.dm

```diff
@@ -0,0 +1,158 @@
+	rapid_melee = 3
+	move_to_delay = 2
+	damage_coeff = list( RED_DAMAGE = 0.9, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 0.5, PALE_DAMAGE = 1.5 )
+	melee_damage_lower = 5
+	melee_damage_upper = 35 // Idea taken from the old PR, have a large damage range to immitate its fucked rolls and crit chance.
+	melee_damage_type = BLACK_DAMAGE
+	stat_attack = HARD_CRIT
+	attack_sound = 'sound/abnormalities/drifting_fox/fox_melee_sound.ogg'
+	attack_verb_simple = "thwacks"
+	attack_verb_continuous = "thwacks"
+	can_breach = TRUE
+	threat_level = HE_LEVEL
+	start_qliphoth = 2
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = 45,
+		ABNORMALITY_WORK_INSIGHT = 45,
+		ABNORMALITY_WORK_ATTACHMENT = list(25,30,35,40,45),
+		ABNORMALITY_WORK_REPRESSION	= 0,
+	)
+	work_damage_amount = 10
+	work_damage_type = BLACK_DAMAGE
+	abnormality_origin = ABNORMALITY_ORIGIN_LIMBUS
+
+	ego_list = list(
+		/datum/ego_datum/weapon/sunshower,
+		/datum/ego_datum/armor/sunshower
+		)
+	gift_type = /datum/ego_gifts/sunshower
+	gift_message = "The fox plucks an umbrella from its back and gives it to you, perhaphs as thanks?"
+
+	var/list/pet = list()
+	pet_bonus = "yips"
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/funpet(mob/petter)
+	pet += petter
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/WorkChance(mob/living/carbon/human/user, chance, work_type)
+	if(user in pet)
+		if(work_type == ABNORMALITY_WORK_ATTACHMENT)
+			chance += 30
+		return chance
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(user in pet)
+		pet -= user
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(-1)
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
+	if(get_attribute_level(user, TEMPERANCE_ATTRIBUTE) <= 40)
+		datum_reference.qliphoth_change(-1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/BreachEffect(mob/living/carbon/human/user)
+	..()
+	icon_living = "fox_breach"
+	icon_state = icon_living
+	pixel_y = -6
+
+/mob/living/simple_animal/hostile/abnormality/drifting_fox/death(gibbed)
+	density = FALSE
+	animate(src, alpha = 0, time = 10 SECONDS)
+	QDEL_IN(src, 10 SECONDS)
+	..()
+
+/mob/living/simple_animal/hostile/abnormality/AttackingTarget(atom/attacked_target)
+	. = ..()
+	if(!ishuman(target))
+		return..()
+	var/mob/living/carbon/human/L = target
+	L.apply_status_effect(STATUS_EFFECT_FALSEKIND)
+	return ..()
+
+/datum/status_effect/false_kindness // MAYBE the black sunder shti works this time.
+	id = "false_kindness"
+	duration = 2 SECONDS //lasts 2 seconds becuase this is for an AI that attacks fast as shit, its not meant to fuck you up with other things.
+	alert_type = /atom/movable/screen/alert/status_effect/false_kindness
+	status_type = STATUS_EFFECT_UNIQUE
```

Comment:
Refresh status may be better here if you want it to keep the effect up as long as it attacks and then for 2 seconds after it stops.